eternalcore-core/src/main/java/com/eternalcode/core/feature/essentials/alert/AlertManager.java

```diff
@@ -0,0 +1,82 @@
+package com.eternalcode.core.feature.essentials.alert;
+
+import com.eternalcode.commons.scheduler.Scheduler;
+import com.eternalcode.core.injector.annotations.Inject;
+import com.eternalcode.core.injector.annotations.component.Service;
+import com.eternalcode.core.notice.EternalCoreBroadcastImpl;
+import com.eternalcode.core.notice.NoticeTextType;
+import com.eternalcode.core.translation.Translation;
+import com.eternalcode.core.viewer.Viewer;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+@Service
+class AlertManager {
+
+    private final Map<AlertKey, List<EternalCoreBroadcastImpl<Viewer, Translation, ?>>> broadcasts = new HashMap<>();
+    private final Scheduler scheduler;
+
+    @Inject
+    AlertManager(Scheduler scheduler) {
+        this.scheduler = scheduler;
+    }
+
+    void addBroadcast(UUID uuid, NoticeTextType type, EternalCoreBroadcastImpl<Viewer, Translation, ?> broadcast) {
+        this.broadcasts.computeIfAbsent(new AlertKey(uuid, type), k -> new ArrayList<>()).add(broadcast);
+    }
+
+    boolean removeBroadcastWithType(UUID uuid, NoticeTextType type) {
+        AlertKey key = new AlertKey(uuid, type);
+        if (this.broadcasts.containsKey(key)) {
+            this.broadcasts.remove(key);
+            return true;
+        }
+        return false;
+    }
+
+    boolean removeLatestBroadcastWithType(UUID uuid, NoticeTextType type) {
+        AlertKey key = new AlertKey(uuid, type);
+        List<EternalCoreBroadcastImpl<Viewer, Translation, ?>> broadcastList = this.broadcasts.get(key);
+
+        if (broadcastList != null && !broadcastList.isEmpty()) {
+            broadcastList.remove(broadcastList.size() - 1);
+            if (broadcastList.isEmpty()) {
+                this.removeBroadcastWithType(uuid, type);
+            }
+            return true;
+        }
+        return false;
+    }
+
+    void clearBroadcasts(UUID uuid) {
+        this.broadcasts.entrySet().removeIf(entry -> entry.getKey().uuid().equals(uuid));
+    }
+
+    void send(UUID uuid) {
+        this.broadcasts.forEach((alertKey, broadcastList) -> {
+            if (!alertKey.uuid().equals(uuid)) {
+                return;
+            }
+
+            NoticeTextType type = alertKey.type();
+            if (type == NoticeTextType.TITLE || type == NoticeTextType.SUBTITLE || type == NoticeTextType.ACTIONBAR) {
+                for (int i = 0; i < broadcastList.size(); i++) {
+                    EternalCoreBroadcastImpl<Viewer, Translation, ?> broadcast = broadcastList.get(i);
+                    scheduler.runLater(broadcast::send, Duration.ofSeconds(i * 2L));
```

Comment:
We should move Duration.ofSeconds(i * 2L) to configuration (you can use `Duration` type in config classes)

Reply:
I would argue that we shouldn't move Duration to configuration, as the current setup ensures that each message displayed on the screen appears in full with a consistent interval.

For example, increasing the interval from 2 to 4 seconds would only create longer pauses between messages, which isn't necessarily needed. The user might get the impression that the system is slow or that messages are appearing at random times. The current interval was chosen to provide a smooth message flow. Therefore, adjusting this value doesn’t bring any meaningful benefits and may lead to unintended behavior.

Reply:
EternalCore is mostly known for customization. You SHOULD put default value for 2 seconds but changing this would be nice for users. I can imagine users wanting to send these messages at once ex. 0.5s or less.